CIPs/cip-50.md

```diff
@@ -0,0 +1,50 @@
+---
+cip: 50
```

Comment:
If you want to overwrite your previous CIP, you can keep 45. Otherwise, let's use 51.